kodilla-testing/src/main/java/com/kodilla/testing/collection/OddNumbersExterminator.java

```diff
@@ -1,4 +1,29 @@
 package com.kodilla.testing.collection;
 
+import java.util.LinkedList;
+import java.util.List;
+
 public class OddNumbersExterminator {
+    private List<Integer> numbers= new LinkedList<>();
+
+
+    public OddNumbersExterminator(List<Integer> numbers) {
+        this.numbers = numbers;
+    }
```

Comment:
zmienna klasowa number jest niewykorzystana i powinna byc usunieta. Jedyna zmienna ktorej uzywasz to zmienna lokalna number z metody exterminate

Reply:
Tak to prawda, ale ja jeszcze używam tej zmiennej do utworzenia obiektu tej klasy evenNumbers, na którym później uruchamiam funkcję exterminate.

Reply:
wrocmy prosze do tego na spotkaniu